docs/fundamentals/wadm/deploying.md

```diff
@@ -2,13 +2,211 @@
 title: "Deploying Wadm"
 date: 2020-01-19T00:00:00+00:00
 icon: "ti-map" # themify icon pack : https://themify.me/themify-icons
-description: "Deploying and Running the Wadm Application in a Cluster"
+description: "Deploying and Running the Wadm Application"
 type: "docs"
 sidebar_position: 1
 ---
 
-This document will guide you through the steps and configuration necessary to deploy the [wadm](https://github.com/wasmcloud/wadm) application in a lattice.
```

Comment:
Let's add a note at the top that during the developer loop, wash will run wadm for you, and this page is for running wadm in "prod"